src/Sfx/package.json

```diff
@@ -45,13 +45,16 @@
     "angular-sanitize": "1.5.0",
     "angular-ui-bootstrap": "2.5.6",
     "bootstrap-sass": "3.4.1",
+    "cytoscape": "^3.11.0",
+    "cytoscape-popper": "^1.0.4",
     "d3": "3.5.17",
     "jquery": "^3.4.0",
     "jquery-ui": "^1.12.1",
     "lodash": "^4.17.15",
     "moment": "^2.22.2",
     "nouislider": "^14.0.2",
     "path": "^0.12.7",
+    "tippy.js": "^2.6.0",
```

Comment:
what is tippy for?

Reply:
this is to popup a tooltip when clicking on one of those nodes